src/ai/backend/client/cli/session.py

```diff
@@ -926,16 +933,15 @@ def watch(session_name_or_id: str, owner_access_key: str, scope: str, max_wait:
         session_names = _fetch_session_names()
         if not session_names:
             click.echo('No matching items.')
```

Comment:
I think this echo should target stderr, not to confuse the JSON parse when `--output=json` is used.